docs/expected_loss.ipynb

```diff
@@ -11,7 +11,7 @@
     "\n",
     "We are now interested in the expected loss $E(\\text{loss})$ a user suffers due to that. Loss is defined as losing access to their funds. In the federated setting that can be either due to theft or due to the adversary stopping the federation (which is easier). There exist countermeasures against adversaries merely stopping the federation (e.g. for ransom) but we will pessimistically assume that both failure modes mean loss of funds for the user (even if only for a limited time). In the centralized custodial case on the other hand the money gets stolen or not.\n",
     "\n",
-    "We explore both federations of optimal sizes $(i \\cdot 3) + 1$ for as well as federations of suboptimal sizes. We see that adding members to a federation of already optimal size hurts resilience in our model since there are more potentially malicious parties but the threshold for how many malicious parties the system can handle does not change. This particular optimality number stems from the fact that we consider frozen funds as lost (since there is potential for blackmail etc.). This happens when the liveness of our consensus algorithm [HBBFT](https://eprint.iacr.org/2016/199.pdf) is no longer guaranteed."
+    "We explore both federations of optimal sizes $(i \\cdot 3) + 1$ for as well as federations of suboptimal sizes. We see that adding members to a federation of already optimal size hurts resilience in our model since there are more potentially malicious parties but the threshold for how many malicious parties the system can handle does not change. This particular optimality number stems from the fact that we consider frozen funds as lost (since there is potential for blackmail etc.). This happens when the liveness of our consensus algorithm [AlephBFT](https://arxiv.org/pdf/1908.05156.pdf) is no longer guaranteed."
```

Comment:
Seems a little odd to change this since this was done when HBBFT was in the code, but I guess its ok